ci/pipelines/deploy-to-test.yml

```diff
@@ -1422,9 +1429,18 @@ jobs:
           AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
           AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
           AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
-      - put: adminusers-latest-ecr-registry-test
-        params:
-          image: adminusers-candidate-ecr-registry-test/image.tar
+      - in_parallel:
+        - do:
+          - put: adminusers-ecr-registry-test
+            params:
+              image: adminusers-candidate/image.tar
+              additional_tags: parse-candidate-tags/release-tag
+          - put: adminusers-latest
+            params:
+              image: adminusers-candidate/image.tar
+        - put: adminusers-dockerhub
+          params:
+              image: adminusers-candidate/image.tar
 #    on_failure:
```

Comment:
Are we ready to un-comment these notifications yet?

Reply:
Yeah I think we need to but I wanted to test this before I annoyed everyone :D Will do it in this PR now it's tested :)

Reply:
Added 👍 thanks :)